spec/controllers/api/v1/packages_controller_spec.rb

```diff
@@ -519,6 +519,55 @@ def test_packages_location_changes(package)
       expect(subject['items'][1]['inventory_number']).to eql("456222")
       expect(subject['items'][2]['inventory_number']).to eql("456111")
     end
+
+    it 'should find items by notes' do
+      create :package, received_quantity: 1, notes: "butter"
+      create :package, received_quantity: 1, notes: "butterfly"
+      create :package, received_quantity: 1, notes: "margarine"
+      get :search_stockit_items, searchText: "UTter", showQuantityItems: 'true'
+      expect(response.status).to eq(200)
+      expect(subject['meta']['total_pages']).to eql(1)
+      expect(subject['meta']['search']).to eql("UTter")
+      expect(subject['items'].length).to eql(2)
+      expect(subject['items'][0]['notes']).to eql("butter")
+      expect(subject['items'][1]['notes']).to eql("butterfly")
+    end
+
+    it 'should find items by designation_name' do
+      create :package, received_quantity: 1, designation_name: "pepper"
+      create :package, received_quantity: 1, designation_name: "peppermint"
+      create :package, received_quantity: 1, designation_name: "garlic"
+      get :search_stockit_items, searchText: "peP", showQuantityItems: 'true'
+      expect(response.status).to eq(200)
+      expect(subject['meta']['total_pages']).to eql(1)
+      expect(subject['meta']['search']).to eql("peP")
+      expect(subject['items'].length).to eql(2)
+      expect(subject['items'][0]['designation_name']).to eql("pepper")
+      expect(subject['items'][1]['designation_name']).to eql("peppermint")
+    end
+
+    it 'should find items by state' do
+      pkg = create :package, received_quantity: 1, state: 'missing'
+      create :package, received_quantity: 1, state: 'expecting'
+      get :search_stockit_items, searchText: "missing", showQuantityItems: 'true'
```

Comment:
Can you remove `showQuantityItems: 'true'` from here and let me know if it works or not.

Thanks.

Reply:
All the items created for the test have a quantity of 1
I tested to make sure, working fine  :)